src/utils/management/commands/clean_up_amps.py

```diff
@@ -0,0 +1,22 @@
+from submission.models import Article
+from re import sub
+from tqdm import tqdm
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Resaves articles that have lots of of escaped ampersands in their abstracts.
+    These extra escaped ampersands are created by Python bleach when an ampersand
+    is found inside an HTML comment. An earlier version of Janeway set
+    BLEACH_STRIP_COMMENTS to False in global settings, but it is now True.
+    So all we need to do is re-save the articles.
+    """
+
+    help = "Removes commented markup and long strings of ampersands from article abstracts."
+
+    def handle(self, *args, **options):
+        articles = Article.active_objects.filter(abstract__contains="&amp;amp;")
+        for article in tqdm(articles):
+            article.save()
```

Comment:
This command has a pre-condition on `BLEACH_STRIP_COMMENTS` being set to `True`

We should handle the case of the environment being set as `BLEACH_STRIP_COMMENTS = False` by either:

A. Strip repeated ampersands regardless of the config of `BLEACH_STRIP_COMMENTS` (and perhaps warn the user)
B. Assert `BLEACH_STRIP_COMMENTS` is set to `True` ahead of running the command